src/components/Banner/BannerContextManager.js

```diff
@@ -10,12 +10,23 @@ import React, {useState, useLayoutEffect} from 'react';
 import BannerContext from './BannerContext';
 
 let activeBanner = null;
+let snooze_start_date = null;
+
+const today = new Date();
+function addTimes(date, days) {
+  const time = new Date(date);
+  const newTime = time.setDate(time.getDate() + days);
```

Comment:
I think dates are mutable so you don’t need the return value. It’s `time` itself that gets modified by the call.